### ISSUE TITLE
ADDON IDEA: email

### ISSUE THREAD

**Issue by dj-hedgehog** [artificialio/sane#92]
An E-Mail Addon which lets user easily send mails through their SANE-applications.

On the server-side I'd implement a MailController using https://github.com/andris9/Nodemailer.

On the client-side my imagination ends with a 'ready-to-use' contact-form component but this won't suit everyone's needs. Do you have better ideas how it could be made more suitable to the users different needs?

**Comment by mgenev** [comment]
I like this a lot too, I would think the easiest way to do this is by using http://node-machine.org/machinepack-mandrill or http://node-machine.org/machinepack-mailgun

**Comment by Globegitter** [comment]
Yep that is a good idea - there is also now https://github.com/balderdashy/sails-hook-email

I will think about if there are any other good use-cases for the frontend. But having a `contac-form` component within ember does sound nice.
